spec.bs

```diff
@@ -654,8 +655,9 @@ To <dfn>generate and score bids</dfn> given an [=auction config=] |auctionConfig
         1. Let |trustedBiddingSignals| be an [=ordered map=] whose [=map/keys=] are [=strings=] and
           whose [=map/values=] are {{any}}.
         1. [=list/For each=] |key| of |ig|'s [=interest group/trusted bidding signals keys=]:
-          1. If |allTrustedBiddingSignals|[|key|] [=map/exists=], then [=map/set=]
-            |trustedBiddingSignals|[|key|] to |allTrustedBiddingSignals|[|key|].
+          1. If |allTrustedBiddingSignals| is not failure and |allTrustedBiddingSignals|[|key|]

```

Comment:
failure -> null.
https://github.com/WICG/turtledove/blob/main/spec.bs#L858 returns << null, null >> when there's an error, not failure.